src/main/java/com/amplitude/HttpCall.java

```diff
@@ -87,7 +87,7 @@ protected Response syncHttpCallWithEventsBuffer(List<Event> events)
       // Also SocketTimeoutException, when the HTTP request times out.
       JSONObject timesOutResponse = new JSONObject();
       timesOutResponse.put("status", Status.TIMEOUT);
```

Comment:
Should this be `Status.UNKNOWN` ?

Reply:
I've decided to use the TIMEOUT status for all of these errors/cases. In the documentation, it says that TIMEOUT is used for any server side or client side HTTP request that times out. This is the best way to organize it I think, let me know if you had more context or ideas

Reply:
Yea, but I looked at the ticket in Jira, it says the test case will return UNKNOW in the response.

Reply:
Yeah, that's why the number was changed to 408. So it should point to TIMEOUT

Reply:
Oh, I see.